charts/argo-workflows/templates/controller/workflow-controller-servicemonitor.yaml

```diff
@@ -3,8 +3,10 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ template "argo-workflows.controller.fullname" . }}
-  {{- with .Values.controller.serviceMonitor.namespace }}
-  namespace: {{ . }}
+  {{- if .Values.controller.serviceMonitor.namespace }}
+  namespace: {{ .Values.controller.serviceMonitor.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace | quote }}
   {{- end }}
```

Comment:
This can be simplified on one line without an if/else:
```
  namespace: {{ default .Release.Namespace .Values.controller.serviceMonitor.namespace | quote }}
```

Reply:
Thank you for reviewing! I fixed on https://github.com/argoproj/argo-helm/pull/1956/commits/9c308cd7f3ae8557d694e2b5c676d3fb2b2817ed . 🙋